Updated psimol/configs/atom_properties.yml

macbiel opened the pull request:
Corrected and extended covalent radii
Confirmed metallic radii
Corrected valences
Confirmed atomic masses

Commit:
Updated psimol/configs/atom_properties.yml

dav3794 commented on psimol/configs/atom_properties.yml:
```diff
@@ -80,8 +83,9 @@ O:
-    - 0.60 # double bond
+    - 0.63
+    - 0.57
+    - 0.53
```
```suggestion

```
Let's skip triple bonded oxygens as unusual cases.

dav3794 commented on psimol/configs/atom_properties.yml:
```diff
@@ -138,18 +150,20 @@ Si:
   mass: 31.0
   atomic_number: 15
-  valence: 5
+  valence: 3
```
```suggestion
  valence: 5
```
Phosphorus is organically usually at V oxidation state (in ATP, DNA etc).

dav3794 commented on psimol/configs/atom_properties.yml:
```diff
@@ -158,8 +172,9 @@ S:
-    - 1.02 # double bond
+    - 1.03
+    - 0.94
+    - 0.95
```
```suggestion
```
Similarly as with oxygen, skip unusual triple bonded sulfur, as it's uncommon in organic molecules.

dav3794 commented:
Basically we can remove all cases of triple bondings which have higher or equal covalent radius than double bond - these are usually obtained in specific conditions (as it is physically abnormal to have longer triple than double bond) and we cannot really handle such things correctly while parsing xyz coordinates. 

+ Most importantly, triple bonds for elements from groups 1-2 and 16-17 should be removed. We could also remove double bonds for group 1.

dav3794 commented:
Additionally, we could possibly add remaining atoms up to bizmuth (so we would have gold, platinum, mercury and lead supported as well), but we can skip the f-block.

Commit:
Updated psimol/configs/atom_properties.yml

macbiel commented:
Removed overlong triple bonds from all elements
Removed triple bonds from elements of groups 1, 2, 16 & 17
Removed double bonds from elements of group 1
Extended the list to include elements of period 6 sans lanthanides, i.e.: Cs, Ba, and Lu to Rn

dav3794 reviewed (approved):
Splendid

dav3794 committed:
Update psimol/configs/atom_properties.yml
